packages/fiori/src/themes/Wizard.css

```diff
@@ -65,7 +65,7 @@
 .ui5-wiz-nav {
 	box-sizing: border-box;
 	height: 4rem;
-	padding: 0.875rem 1rem;
+	padding: 0.875rem 2rem;
```

Comment:
By default it is 1rem

Reply:
setting it to 2 keeps the content padding aligned with the header part, I would keep it 2.